packages/repository/src/model.ts

```diff
@@ -40,6 +40,7 @@ export type PropertyType =
 export interface PropertyDefinition {
   type: PropertyType; // For example, 'string', String, or {}
   id?: boolean | number;
+  hidden?: boolean;
```

Comment:
Please add a TSDoc comment to explain the purpose of this property and mention how it interacts with model-level setting `hiddenProperties`.

```suggestion
  /**
   * (put the text here)
   */
  hidden?: boolean;
```